core/store/migrations/0040_create_job_proposals.go

```diff
@@ -0,0 +1,36 @@
+package migrations
+
+import (
+	"gorm.io/gorm"
+)
+
+const up40 = `
+CREATE TABLE job_proposals (
+    id BIGSERIAL PRIMARY KEY,
+	spec TEXT NOT NULL,
+	status VARCHAR (50) NOT NULL,
```

Comment:
There is not too much benefit in using a VARCHAR or smaller length in postgres, other than for sanity checking. There's no performance or space saving to be had.

Is it static enough that an ENUM would suit here?

Reply:
The statuses will be `pending`, `approved`, and `rejected`. However we may need to add more in the future so I would prefer not to use an ENUM.

Reply:
text is fine